samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/JSON.java

```diff
@@ -64,7 +64,8 @@ public Class getClassForElement(JsonElement readElement) {
             }
           })
         ;
-        return fireBuilder.createGsonBuilder();
+        GsonBuilder builder = fireBuilder.createGsonBuilder();
```

Comment:
why this change?

Reply:
This is a change related to one of my other PRs, not sure why it was included here, removed after regenerating samples. Sorry about that.